Week2/Mohammad-Darwesh-JS2-week2-homework/maartjes-work.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
```

Comment:
Have you tried to test your homework using `npm run test-week2`? The test is looking at the files in the `homework` folder, so instead of putting new files in a new folder you should edit the files there.

Reply:
Hello Paulbremer,
Thank you for your feedback .
I did just what you tell me and it pass the test just as you said .
There is some changing in json files ....i realy do know why but i did push anyway.
thanks,

Reply:
Looking good @mohammadnaefaldarwesh the tests are passing fine now! 👌